privacyidea/static/components/token/views/token.import.html

```diff
@@ -124,4 +139,8 @@ <h3 translate>Import tokens from a file</h3>
     <div translate>
         Uploaded Tokens: {{ uploadedTokens }}
     </div>
+    <div>
+        <!--        How to translate this?-->
+        Skipped Tokens: {{ notImportedTokens }}
```

Comment:
You can either do:

~~~~html
<div translate>
    Skipped Tokens: {{ notImportedTokens }}
</div>
~~~~

Or 

~~~~html
<div>
    <translate>Skippted Tokens<translate>: {{ notImportedTokens }}
</div>
~~~~

I think the 2nd is better.